sunpy/database/database.py

```diff
@@ -954,6 +955,14 @@ def redo(self, n=1):
         """
         self._command_manager.redo(n)  # pragma: no cover
 
+    def display_entries(self, columns=None, sort=False):
+        print display_entries(self, columns, sort)
+        return
+
+    def show_in_browser(self, columns=None, sort=False):
```

Comment:
you should expose the `jsviewer` kwarg in the method definition so users can change it.